docs/src/IconButton.doc.js

```diff
@@ -12,202 +12,357 @@ const card = c => cards.push(c);
 card(
   <PageHeader
     name="IconButton"
-    description="The IconButton component allows you to define an action with a specific Icon."
+    description="IconButton allow users to take actions and make choices with a single click on a specific Icon. They are typically found in forms, dialog, and toolbars. Some buttons are specialized for particular tasks, such as navigation or presenting menus."
```

Comment:
@AlbertCarreras 
* `allow` => `allows`
* The `They` in the second sentence refers to a single `<IconButton />`, could we update that?

Reply:
DONE